.travis.yml

```diff
@@ -1,28 +1,35 @@
 language: php
 
 php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
-    - hhvm
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
+  - hhvm
```

Comment:
please don't change the indentation of the file. It makes things much harder to review